pom.xml

```diff
@@ -638,6 +638,26 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
how's java-cloud-bom being used? typically plugins and common build functionalities do not go into a bom.

Reply:
This is being imported into `cloud.google.com:libraries-bom`. The build configuration is necessary to release this `pom.xml`. It's not (at least should not be) used as any project's parent.

Reply:
got it, then adding this plugin probably won't have any real/intended effects.

Reply:
let's sync up w/ stephanie and ben to discuss the details?
also, there is a current doc on the design of our build/pom relatoinships?